Firebase/Core/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v6.0.1 -- M48
+- [changed] `FirebaseApp` name accepts any alpha-numeric character not only ASCII (#2609)
```

Comment:
Nit: this should be in past tense, something like:
"Allowed `FirebaseApp` name to accept any alpha-numeric character instead of only ASCII (#2609)"